sourmash/_minhash.py

```diff
@@ -544,13 +544,14 @@ def __iadd__(self, other):
 
     merge = __iadd__
 
-    def set_abundances(self, values):
+    def set_abundances(self, values, no_clear=False):
```

Comment:
calling it `no_clear` leads to a lot of double negatives in the code. How about using
```suggestion
    def set_abundances(self, values, clear=True):
```
here, and adapting the `if not no_clear:` below to `if clear:`?